app/src/main/java/dev/randombits/intervaltimer/SettingsFragment.kt

```diff
@@ -25,35 +27,42 @@ class SettingsFragment : Fragment() {
     }
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
+        super.onCreate(savedInstanceState);
 
         arguments?.let {
-            activeTime = it.getInt(ARG_PARAM1)
-            restTime = it.getInt(ARG_PARAM2)
+            activeTime = it.getInt(ARG_PARAM1);
+            restTime = it.getInt(ARG_PARAM2);
         }
     }
 
+    @SuppressLint("SetTextI18n")
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View? {
+    ): View {
         val view = inflater.inflate(R.layout.fragment_settings, container, false) as View;
 
-        activeInput = view.findViewById(R.id.activeTime)
-        activeInput!!.setText(activeTime.toString())
-        restInput = view.findViewById(R.id.restTime)
-        restInput!!.setText(restTime.toString())
+        activeInput = view.findViewById(R.id.activeTime);
+        activeInput!!.setText(activeTime.toString());
+        restInput = view.findViewById(R.id.restTime);
+        restInput!!.setText(restTime.toString());
 
         view.findViewById<View>(R.id.activeTime_less).setOnClickListener {
             val activeTime = activeInput!!.text.toString();
             if (activeTime.isBlank()) {
                 activeInput!!.setText("45");
             } else {
-                if (Integer.parseInt(activeTime) >= 5) {
-                    activeInput!!.setText((Integer.parseInt(activeTime) - 5).toString());
+                val newActiveTime = Integer.parseInt(activeTime) - 5
+                if (newActiveTime >= 5) {
+                    activeInput!!.setText(newActiveTime.toString())
+                } else {
+                    Toast.makeText(
```

Comment:
The user can still enter numbers via the keyboard without using the buttons, so this doesn't completely prevent them from entering 0 here. Do you know if there is a way to set a minimum value of 1 on the input? Otherwise, we could allow entering 0, but display a toast error when they try to click the start button.